### PR TITLE
Add Python 3.6

### PR DESCRIPTION
**Pull request by gustavla**
This adds support for 3.6. I don't know much about managing conda packages, so I have not tested this locally yet.

### CODE REVIEW

**Comment by basnijholt** [comment]
The only thing that is needed is forking the repo, run `conda-smithy rerender`, commit, push, and open a PR.

It looks like you did that 😄

**Comment by gustavla** [comment]
@basnijholt I did all those steps except the conda smithy one, which I didn't know about. Not sure if that means I missed an important update to any of the files. If it passes the CI it should be fine though, right? (Once Travis wakes up).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```
Comment (ocefpaf) [inline recipe/meta.yaml:15]: There is no need for this when adding another item on the build matrix. All the build number `1` for Python `2.7` and `3.5` will be identical to build number `0`. (But it does not hurt to leave it there.)

**Comment by ocefpaf** [comment]
Considering how slow Travis-CI is lately and the simplicity of this PR I would be OK if you guys want to merge to get the Windows and Linux packages up soon.

**Pull request merged**